stdlib/number.gr

```diff
@@ -660,3 +660,89 @@ provide let toRadians = degrees => degrees * (pi / 180)
  * @since v0.5.4
  */
 provide let toDegrees = radians => radians * (180 / pi)
+
+/**
+ * Clamps a number to a given range.
+ *
+ * @param range: The inclusive range to clamp within.
+ * @param input: The number to clamp
+ * @returns The clamped number
+ *
+ * @since v0.6.0
+ */
+provide let clamp = (range, input) => {
+  if (isNaN(input)) {
+    input
+  } else {
+    let rangeEnd = max(range.rangeStart, range.rangeEnd)
+    let rangeStart = min(range.rangeStart, range.rangeEnd)
+
+    if (input > rangeEnd) rangeEnd else if (input < rangeStart) rangeStart
+    else input
+  }
+}
+
+/**
+ * Linearly interpolates across the provided range by the given amount.
+ *
+ * @param range: An inclusive Range to interpolate between
```

Comment:
```suggestion
 * @param range: An inclusive range to interpolate between
```
Don't need to capitalize because the docs will have the type capitalized